### DISCUSSION TITLE
Problems installing surface core on Surface pro 7

### DISCUSSION THREAD

**Discussion by stellankinberg** [linux-surface/linux-surface#492] [Q&A]
I tried invane to follow the instructions for Ubuntu in 
https://github.com/linux-surface/linux-surface/wiki/Installation-and-Setup
I have the latest version of Ubuntu.
1 Can not add key to ubuntu
2. I can not  add repository in ubuntu sources

These two commands do not work from my home dir:

$ wget -qO - https://raw.githubusercontent.com/linux-surface/linux-surface/master/pkg/keys/surface.asc \
    | gpg --dearmor | sudo dd of=/etc/apt/trusted.gpg.d/linux-surface.gpg
After this you can add the repository configuration and update APT.

$ echo "deb [arch=amd64] https://pkg.surfacelinux.com/debian release main" \
	| sudo tee /etc/apt/sources.list.d/linux-surface.list

**Comment by qzed** [comment]
What's the error with the commands? Did you accidentally copy the `$` sign?
